ports/espressif/boards/adafruit_esp32s2_camera/pins.c

```diff
@@ -0,0 +1,67 @@
+#include "py/objtuple.h"
+#include "shared-bindings/board/__init__.h"
+#include "shared-module/displayio/__init__.h"
+
+STATIC const mp_rom_obj_tuple_t camera_data_tuple = {
+    {&mp_type_tuple},
+    8,
+    {
+        MP_ROM_PTR(&pin_GPIO13),
+        MP_ROM_PTR(&pin_GPIO15),
+        MP_ROM_PTR(&pin_GPIO16),
+        MP_ROM_PTR(&pin_GPIO14),
+        MP_ROM_PTR(&pin_GPIO12),
+        MP_ROM_PTR(&pin_GPIO10),
+        MP_ROM_PTR(&pin_GPIO9),
+        MP_ROM_PTR(&pin_GPIO7),
+    }
+};
+
+
+STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
+    CIRCUITPYTHON_BOARD_DICT_STANDARD_ITEMS
```

Comment:
Is this board in the board repo? I didn't see it.

Reply:
Development\Dev Boards\BrainCraft\Pylaroid